pyproject.toml

```diff
@@ -8,12 +8,13 @@ authors = []
 python = "^3.7"
 flask = "^1.1.2"
 python-dotenv = "^0.14.0"
-requests = "^2.25.1"
 gunicorn = "^20.0.4"
 virtualenv = "^20.4.3"
 pytest-watch = "^4.2.0"
 pytest = "^6.2.2"
 selenium = "^3.141.0"
+pymongo = {extras = ["srv"], version = "^3.12.0"}
+mongomock = "^3.23.0"
```

Comment:
Minor, but we could change this to a dev-dependency (either by installing with `poetry add --dev <library>` or by moving this line into the section below) which will save your app from always installing this library when just being set up for production (this is useful for e.g. restricting the attack surface of dependencies in production)